frontend/shared/src/utils/string.utils.ts

```diff
@@ -23,3 +23,6 @@ export const getBooleanValueFromString = (value?: string): boolean | null => {
   }
   return true;
 };
+
+export const isEmpty = (value?: string): boolean =>
+  typeof value === 'string' ? value?.trim().length === 0 : Boolean(!value);
```

Comment:
The idea was to check the type runtime, then you need this `typeof value === 'string'`. But actually this method is obsolete because in later prs things get more complicate and I just replaced this with lodash isEmpty function.